lib/api/funnel.js

```diff
@@ -263,6 +264,21 @@ class Funnel {
       return 1;
     }
 
+    const httpCfg = global.kuzzle.config.http;
+    if (request.input.headers
+      && has(request.input.headers, 'origin')
+      && !global.kuzzle.config.internal.allowAllOrigins
+      && !httpCfg.accessControlAllowOrigin.includes(request.input.headers.origin) // If the domain is not present in the list
```

Comment:
Optional Nit
```suggestion
    const httpConfig = global.kuzzle.config.http;
    if (request.input.headers
      && has(request.input.headers, 'origin')
      && !global.kuzzle.config.internal.allowAllOrigins
      && !httpConfig.accessControlAllowOrigin.includes(request.input.headers.origin) // If the domain is not present in the list
```

Reply:
I saw that we are using the `httpCfg` style in other places (protocol), you can ignore this comment if you think httpCfg > httpConfig.